src/locales/en.js

```diff
@@ -1,7 +1,9 @@
 import { add } from "@ckeditor/ckeditor5-utils/src/translation-service.js";
 
 const translations = {
+    "Equation preview": "Preview",
     "Display mode": "Line break",
+    "Insert math": "Formula",
```

Comment:
'Insert equation in TeX format' is already ok? I would keep the same keys in all translations, for consistency

Reply:
good idea, added this key in en as well, also when not changed for now